meson.build

```diff
@@ -151,6 +151,11 @@ if dl.found()
     netatalk_common_link_args += '-ldl'
 endif
 
+math = cc.find_library('m', required: true)
```

Comment:
What is this library for?

Reply:
The dh.c file in WolfSSL is linked to the external math library.

When you use the math library with all recent versions of GCC on GNU/Linux systems you have to explicitly link to it. It is not automatically linked to along with the rest of the standard C library.